appveyor.yml

```diff
@@ -0,0 +1,30 @@
+# appveyor file
+# http://www.appveyor.com/docs/appveyor-yml
+
+init:
+  - git config --global core.autocrlf input
+
+# what combinations to test
+environment:
+  matrix:
+    #node.js
+    - nodejs_version: 0.10
+    - nodejs_version: 0.11
+    - nodejs_version: 0.12
+    #io.js
+    - nodejs_version: 2.5.0
+
+install:
+  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version)
```

Comment:
To avoid the annoying Appveyor network issues when installing node, use `Install-Product` instead. See an example in the SDK: https://github.com/resin-io/resin-sdk/blob/master/appveyor.yml#L15.